python/docs/guide.md

```diff
@@ -181,9 +181,9 @@ You can configure
 for memory allocations using their by configuring the current
 allocator.
 
-```python
-from rmm.allocators.torch import rmm_torch_allocator
-import torch
+  ```python
```

Comment:
*question*: Why the indentation of this whole block by a single character? I think it is unnecessary.

Reply:
Did I do thaaaat? :)